packages/postcss/package.json

```diff
@@ -42,6 +42,7 @@
   "dependencies": {
     "@unocss/config": "workspace:*",
     "@unocss/core": "workspace:*",
+    "@unocss/preset-mini": "workspace:*",
```

Comment:
I think we should avoid introducing this, as well as in the transformer directives